adapters/sovrn/sovrn.go

```diff
@@ -61,6 +39,54 @@ func (s *SovrnAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapt
 		}
 	}
 
+	errs := make([]error, 0, 1)
+	var err error
+	requestImpCopy := request.Imp
```

Comment:
The word `Copy` in this name is misleading, since this isn't a deep copy.

Reply:
Why is this not a deep copy?

Reply:
From what I can tell, this is a deep copy.

Reply:
It is a copy of a struct for primitives only, see https://flaviocopes.com/go-copying-structs/#deep-copy-a-struct an `Imp` definitely has other complex types that aren't deep copied. This has me back off of the `Copy` wording because it's shallow, but it's not deep 😄

Reply:
Thanks, that is exactly what I was not able to find myself.